com.surfshark.Surfshark.metainfo.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+    <id>com.surfshark.Surfshark</id>
+
+    <name>Surfshark</name>
+    <summary>Connect to secure, private web!</summary>
+    <url type="homepage">https://surfshark.com/</url>
+    <url type="contact">https://support.surfshark.com/</url>
+
+    <metadata_license>CC-BY-SA-4.0</metadata_license>
+    <project_license>LicenseRef-proprietary=https://surfshark.com/terms-of-service</project_license>
+    <developer id="com.surfshark">
+        <name>Surfshark B.V</name>
+    </developer>
+    <content_rating type="oars-1.1" />
+    <screenshots>
+        <screenshot>
+            <caption>Surfshark interface</caption>
+            <image type="source">https://raw.githubusercontent.com/flathub/com.surfshark.Surfshark/master/screenshots/screenshot_main.png</image>
+        </screenshot>
+        <screenshot>
+            <caption>Static locations</caption>
+            <image type="source">https://raw.githubusercontent.com/flathub/com.surfshark.Surfshark/master/screenshots/screenshot_static_locations.png</image>
+        </screenshot>
+        <screenshot>
+            <caption>Multihop locations</caption>
+            <image type="source">https://raw.githubusercontent.com/flathub/com.surfshark.Surfshark/master/screenshots/screenshot_multihop_locations.png</image>
```

Comment:
We need working links for this for the build to pass. You can switch the URLs later when this is accepted.

For now you have to use https://raw.githubusercontent.com/flathub/flathub/42ae054b049c92575f1161302ecb220f235c4ec7/screenshots/screenshot_main.png etc.

Reply:
updated